### DISCUSSION TITLE
Copilot doesn't automatically create all necessary functions.

### DISCUSSION THREAD

**Discussion by 14r14** [community/community#6822] [Copilot Conversations]
I was trying to create a function for a program(Python) I'm working on, and when copilot gave me the code, I noticed that it doesn't automatically create other functions that your program uses, and you need to name define it. Is this a known issue, or...?

**Comment by Urethramancer** [comment]
I've noticed the same both for Python and Go. It likes to invent methods/dunctions I don't have sometimes. Ctrl+enter may sometimes suggest a full list of functions though.
